test/basic_app/splat.yaml

```diff
@@ -14,20 +14,24 @@ options:
   asset_path: split/assets
   asm_generated_by: False
 segments:
-  - name: basic_app
+  - name: header
+    type: header
+    start: 0x00
+  - name: dummy_ipl3
     type: code
-    start: 0x00000000
-    vram:  0x00400000
-    bss_size: 0x24
-    subalign: 4
+    start: 0x40
+    vram: 0xA4000040
     subsegments:
```

Comment:
This yaml should specify `compiler: GCC` to generate the right main.data.s and main.rodata.s files, in addition to migrating the jump table rodata into the function that owns it.

Reply:
Fixed